src/settings/MappingProfiles/ViewMappingProfile.js

```diff
@@ -89,6 +89,7 @@ export class ViewMappingProfile extends Component {
         records: PropTypes.arrayOf(PropTypes.object),
       }),
     }).isRequired,
+    parentResources: PropTypes.shape({ marcFieldProtectionSettings: PropTypes.shape({ records: PropTypes.arrayOf(marcFieldProtectionSettingsShape) }) }).isRequired,
```

Comment:
Looks like `marcFieldProtectionSettings` and `records` should be required
```suggestion
    parentResources: PropTypes.shape({
        marcFieldProtectionSettings: PropTypes.shape({
            records: PropTypes.arrayOf(marcFieldProtectionSettingsShape).isRequired,
        }).isRequired,
    }).isRequired,
```